.github/workflows/build.yml

```diff
@@ -307,8 +307,7 @@ jobs:
             libboost-dev, libboost-system-dev, libboost-filesystem-dev,
             libssl-dev,
             rapidjson-dev,
-            libbenchmark-dev,
-            libavif-dev
+            libbenchmark-dev
```

Comment:
I'm going to use the `libavif` from conan on all platforms once `libavif` 1.0.1 gets merged.